spec/models/miq_product_feature_spec.rb

```diff
@@ -112,20 +112,18 @@ def traverse_product_features(product_feature, &block)
     it "returns only visible features" do
       hierarchical_features
       expect(MiqProductFeature).not_to receive(:sort_children)
-      expect(MiqProductFeature.feature_children("miq_report_widget_admin", false)).to match_array(
-        %w(widget_copy widget_edit))
+      expect(MiqProductFeature.feature_children("miq_report_widget_admin", false)).to match_array(%w(widget_copy widget_edit))
```

Comment:
Commits are cheap and make it clearer so I don't look at the same line for 30 seconds trying to find a change.  😉 

But, I"m mostly just :trollface: